digdag-core/src/main/java/io/digdag/core/workflow/WorkflowExecutor.java

```diff
@@ -104,6 +104,8 @@
  *       : ERROR with error
  *
  *   taskSucceeded:
+ *     (if CANCEL_REQUESTED flag is set) lockedTask.setToCanceled:
```

Comment:
This is outside the scope of this PR, but I'm wondering a successfully finished task should be handled as SUCCESS not as CANCELED even if it's received a cancel request...